graphql/schema/types/task.graphql

```diff
@@ -73,6 +73,9 @@ type Task {
   details: TaskEndDetail
```

Comment:
this is just a sanity checking question, but does anything need to be done to usages of `baseStatus` or are we OK there?

Reply:
Nothing needs to change to accomplish the goal of the ticket. I added a comment to clarify that it represents the displayStatus.